bin/run-e2e-tests.sh

```diff
@@ -0,0 +1,11 @@
+# Exit if any command fails
+set -e
+
+# Change to the expected directory
+cd "$(dirname "$0")/../"
+
+# Setup local environement
+( ./bin/setup-local-env.sh )
+
+# Run the tests
+npx cypress run
```

Comment:
`npx` is only bundled with npm 5.x+ which is not explicitly required by our [target Node version](https://github.com/WordPress/gutenberg/blob/master/.nvmrc), though the new Node 8.x / npm 5.x LTS release is supposedly slated to be released by the end of the month. The alternative is:

```
./node_modules/.bin/cypress run
```